frontend/src/views/supp-allowances/SupplementaryFormView.vue

```diff
@@ -64,12 +91,17 @@ export default {
     return {
       loading: false,
       panel: [],
+      models: undefined,
+      clonedModels: [],
     }
   },
   computed: {
     allPageIDs() {
       return this.PAGES?.map((page) => page.id)
     },
+    getSupplementaryTypes() {
+      return SUPPLEMENTARY_TYPES
+    },
```

Comment:
You can avoid defining a computed method by assigning the imported constant in the created() method. See YearyFacilityCost.vue line 200.